src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/mapper/RoboticsDataMapper.java

```diff
@@ -55,13 +55,13 @@ private CaseHeader buildCaseHeader(CaseData caseData) {
 
     private Solicitor buildRespondentSolicitor(CaseData caseData) {
         return Solicitor.builder()
-            .solicitorPartyReference(caseData.getSolicitorReferences().getRespondentSolicitor1Reference())
```

Comment:
solicitor references could be null, and each field inside could also be null

Reply:
done